packages/formatter-markdown/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@hint/formatter-markdown",
+  "version": "1.0.0",
+  "description": "Webhint formatter that creates a Markdown file with the results of the webhint scan.",
+  "main": "dist/src/formatter.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "npm run i18n && npm-run-all build:*",
+    "build-release": "npm run clean && npm run i18n && npm run build:assets && tsc --inlineSourceMap false --removeComments true",
+    "build:assets": "cpx \"./{src,tests}/**/{!(*.ts),.!(ts)}\" dist",
+    "build:ts": "tsc -b",
+    "clean": "rimraf dist",
+    "i18n": "node ../../scripts/create-i18n.js",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint . --cache --ext .js,.md,.ts --ignore-path ../../.eslintignore",
+    "lint:dependencies": "node ../../scripts/lint-dependencies.js",
+    "lint:md": "node ../../scripts/lint-markdown.js",
+    "test": "npm run i18n && npm run lint && npm run build && npm run test-only",
+    "test-only": "nyc ava",
+    "test-release": "npm run i18n && npm run lint && npm run build-release && ava",
+    "watch": "npm run build && npm-run-all --parallel -c watch:*",
+    "watch:assets": "npm run build:assets -- -w --no-initial",
+    "watch:test": "ava --watch",
+    "watch:ts": "npm run build:ts -- --watch"
+  },
+  "nyc": {
+    "extends": "../../.nycrc"
+  },
+  "peerDependencies": {
+    "hint": "^6.0.0"
+  },
+  "repository": {
+    "directory": "packages/formatter-markdown",
+    "type": "git",
+    "url": "https://github.com/webhintio/hint.git"
+  },
+  "dependencies": {
+    "@hint/utils": "^7.0.6",
+    "@hint/utils-fs": "^1.0.5",
+    "@hint/utils-i18n": "^1.0.5",
+    "@hint/utils-types": "^1.1.0",
+    "lodash": "^4.17.20",
+    "fs-extra": "^9.0.1",
+    "moment": "^2.29.0"
```

Comment:
This is a note for the core team:
- We are using version 2.28.0 in another package (maybe that's why `yarn.lock` gets updated?)
- [moment is considered legacy now by their team](https://momentjs.com/docs/#/-project-status/), we should think about moving to other alternatives

Reply:
Maybe we should create an issue to replace moment?

Reply:
Done in https://github.com/webhintio/hint/issues/4044

Reply:
I've seen we'have removed it from the HTML formatted so I've removed it from this formatter.